packages/container/tests/container_test.js

```diff
@@ -554,8 +554,9 @@ QUnit.test('An object with its owner pre-set should be returned from ownerInject
 });
 
 QUnit.test('A deprecated `container` property is appended to every object instantiated from an extendable factory', function() {
+  let owner = { };
   let registry = new Registry();
-  let container = registry.container();
+  let container = owner.__container__ = registry.container({ owner });
```

Comment:
FYI - This is exactly what is done outside of these unit tests.  See [EngineInstance](https://github.com/emberjs/ember.js/blob/f44f1d54ce3ffedc1c41309a9746dbfdad570591/packages/ember-application/lib/system/engine-instance.js#L50-L57) code for confirmation.